docs/source/workflow_inputs.rst

```diff
@@ -2487,13 +2487,17 @@ For air-source HVAC systems with detailed cooling performance data, two or more
   Element                            Type      Units   Constraints  Required  Default    Notes
   =================================  ========  ======  ===========  ========  =========  ==========================================
   ``OutdoorTemperature``             double    F       See [#]_     Yes                  Outdoor drybulb temperature
-  ``Capacity``                       double    Btu/hr  >= 0         Yes                  Cooling capacity at the specified outdoor temperature
+  ``Capacity``                       double    Btu/hr  >= 0         No [#]_              Cooling capacity at the specified outdoor temperature
+  ``CapacityFractionOfNominal``      double    frac    > 0, <= 1    No                   Cooling capacity fraction of nominal at the specified outdoor temperature
```

Comment:
I think this could be simplified by collapsing into a single line with `Capacity and/or CapacityFractionOfNominal` and Required=Yes. You can find other examples in the documentation of doing this. Then you can simplify some of the language in the footnote below.